python/tvm/relax/transform/legalize_ops/common.py

```diff
@@ -39,9 +39,11 @@
 ##################### Utilities #####################
 
 
-def _try_convert_to_scalar_const(expr: Expr) -> Union[Expr, bool, float, int]:
+def _try_convert_to_scalar_const(
+    expr: Expr, python_native: bool = False
+) -> Union[Expr, bool, float, int]:
```

Comment:
Would you mind adding IntImm/FloatImm to the return annotation here, and add some doc string for `python_native`?

Reply:
Also, it would be nice to add this potential problem we found with python_native approach.